testing/src/cypress/e2e/02-institution/07-location-programs.cy.ts

```diff
@@ -58,121 +166,7 @@ describe("Location Program", () => {
-    cy.intercept("GET", "**/location/**").as("location");
-    dashboardInstitutionObject.dashboardButton().click();
-    dashboardInstitutionObject.programsButton().eq(0).click();
-    cy.url().should("contain", "/location-programs");
-    cy.wait("@location");
-    locationProgramObject.firstRowEditButton().click();
-  });
-
-  it.only("Verify that user able to add study period", () => {
-    cy.intercept("GET", "**/education-program/**").as("educationProgram");
-    dashboardInstitutionObject.dashboardButton().click();
-    dashboardInstitutionObject.programsButton().eq(0).click();
-    cy.url().should("contain", "/location-programs");
-    cy.wait("@educationProgram");
-    locationProgramObject.firstRowEditButton().click();
-    locationProgramObject.addStudyPeriodButton().click();
-    locationProgramObject.wait();
-    locationProgramObject
-      .programNameStudyPeriod()
-      .type(programData.studyPeriodName);
-    locationProgramObject.yearOfStudyDropdown().click();
-    locationProgramObject
-      .yearOfStudyDropdownInputText()
-      .type(programData.yearOfStudy)
-      .type("{enter}");
-    locationProgramObject.displayThisToStudentCheckbox().click();
-    locationProgramObject.howWillThisBeOfferedRadioButton().click();
-    locationProgramObject.offeringBeDeliveredRadioButton().click();
-    locationProgramObject.workIntegratedComponentRadioButton().click();
-    locationProgramObject.studyStartDate().type(programData.startDate);
-    locationProgramObject.studyEndDate().type(programData.endDate);
-    locationProgramObject.breakStartDate().type(programData.breakStart);
-    locationProgramObject.breakEndDate().type(programData.breakEnd);
-    locationProgramObject.actualTuitionInput().type(programData.actualTuitions);
-    locationProgramObject
-      .programRelatedInput()
-      .type(programData.programRelatedCosts);
-    locationProgramObject.mandatoryFeesInput().type(programData.mandatoryFees);
-    locationProgramObject
-      .exceptionalExpensesInput()
-      .type(programData.exceptionalExpenses);
-    locationProgramObject.tuitionRemittanceRadioButton().click();
-    locationProgramObject.addStudyPeriodRadioButton().click({ force: true });
-    locationProgramObject.declarationFormForVerification().click();
-  });
-
-  it("Verify that search is working in study period offerings", () => {
-    cy.intercept("GET", "**/education-program/**").as("educationProgram");
-    dashboardInstitutionObject.dashboardButton().click();
-    dashboardInstitutionObject.programsButton().eq(0).click();
-    cy.url().should("contain", "/location-programs");
-    cy.wait("@educationProgram");
-    locationProgramObject.firstRowEditButton().click();
-    locationProgramObject
-      .searchStudyPeriodInput()
-      .type(programData.studyPeriodName)
-      .type("{enter}");
-    locationProgramObject
-      .firstRecordAssertionStudyPeriod()
-      .should("be.visible");
+  it.only("Verify that all credential types exist", () => {
```

Comment:
for my info, what is `.only`

Reply:
This was only testing purposes. It run that particular test ad-hoc from the spec file.